src/utils/logging.ts

```diff
@@ -1,19 +1,11 @@
-import chalk from "chalk";
-
-// eslint-disable-next-line
+// eslint-disable-next-line no-console, @typescript-eslint/no-empty-function
 const log = process.env.NODE_ENV !== "test" ? console.log : () => {};
```

Comment:
Little trick: You can use `() => null` or `() => undefined` if you want to get rid of the ESLint suppression comment.